### PR TITLE
Remove experimental status tag and references to experimental status on website

### PR DESCRIPTION
**Pull request by owenatgov**
Removes the experimental status tag from all docs that use it, it's associated content and references to the experimental status in the tag component and the [develop a component or pattern guidance page](https://design-system.service.gov.uk/community/develop-a-component-or-pattern/).

Closes https://github.com/alphagov/govuk-design-system/issues/3194. More details in issue.

## Extra thoughts
There's a reference to stuff being marked as experimental on the [contribution criteria page](https://design-system.service.gov.uk/community/contribution-criteria/):

>Components and patterns which are not proven usable can be published as experimental. But they must be clearly based on relevant user research from other organisations and best practice, and meet the other criteria.

Should we remove it?

Is removing the reference to the Tag being experimental the right move?

### CODE REVIEW

**Comment by CharlotteDowns** [comment]
Do we also need to remove it from line 80 of contribution-criteria/index.md?

**Comment by CharlotteDowns** [comment]
I totally agree.

Maybe instead of 

> Components and patterns which are not proven usable can be published as experimental. But they must be clearly based on relevant user research from other organisations and best practice, and meet the other criteria.

we say
> Components and patterns which have not yet been proven usable can be published, but they must be clearly based on relevant user research from other organisations, best practice, and meet the other criteria.

Perhaps we can also include a line about the new research guidance.
> We also encourage service teams to [share findings about users](https://design-system.service.gov.uk/community/share-research-findings/) to support contributions.

**Comment by StephenGill** [comment]
I think the issue is that being 'usable' or not isn't binary - it's always specific to a context.

So I'd say something like -

'You can submit a pattern or component as long as you've established that it works in your service. We'll make a judgement about whether it's likely to work in other contexts. And if we publish the pattern or component, we'll be clear about what contexts it's been tried in - and invite feedback about how well it works in different contexts.'

But obviously with less repetition of the word 'context'.

**Comment by owenatgov** [comment]
I've drafted something based on the above suggestions:

>If there isn't a evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.

What do we think?

One thing I'm a little concerned about is that this clashes with one of the other criteria: versatile:

>The implementation is versatile enough that the component or pattern can be used in a range of different services that may need it.

Reading this back, it sorta feels like these two criteria have always contradicted one another. I'm wondering if the context of where it's been tested is the right approach over outlining gaps in testing.

**Comment by CharlotteDowns** [comment]
@owenatgov I'm super happy with:

> If there isn't evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.

(I removed a rogue 'a' before evidence)

On the second point I'm not sure it is meant to contradict. I think what we are trying to get at is, there's an option for things to be published untested if we have good reason to believe that they will be useful and useable in a range of different services once available.

File: src/community/contribution-criteria/index.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Before a new component or pattern is published the working group reviews the imp
       },
       {
         html: '<p>It has been tested  in user research and shown to work with a representative sample of users, including those with disabilities.</p>
-          <p class="govuk-!-margin-bottom-0">Components and patterns which are not proven usable can be published as experimental. But they must be clearly based on relevant user research from other organisations and best practice, and meet the other criteria.</p>'
+          <p class="govuk-!-margin-bottom-0">If there isn\'t evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.</p>'
```
Comment (CharlotteDowns) [inline src/community/contribution-criteria/index.md:80]:
```suggestion
          <p class="govuk-!-margin-bottom-0">If there is not evidence of usability outside of one or a small number of services, the component or pattern can still be published if it meets other contribution criteria. We will be clear about what contexts the component or pattern has been tested in upon publishing and invite feedback about its performance in a wide range of services.</p>'
```

I wonder if @calvin-lau-sig7 can give this a once over?

Comment (calvin-lau-sig7) [reply src/community/contribution-criteria/index.md:80]: Once over'ed! How's this look?

```suggestion
          <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be published, as long as it meets other contribution criteria. We'll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'        
```

Comment (CharlotteDowns) [reply src/community/contribution-criteria/index.md:80]: @owenatgov what do you think? Should I commit this change?

File: src/community/contribution-criteria/index.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Before a new component or pattern is published the working group reviews the imp
       },
       {
         html: '<p>It has been tested  in user research and shown to work with a representative sample of users, including those with disabilities.</p>
-          <p class="govuk-!-margin-bottom-0">Components and patterns which are not proven usable can be published as experimental. But they must be clearly based on relevant user research from other organisations and best practice, and meet the other criteria.</p>'
+          <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be published, as long as it meets other contribution criteria. We\'ll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'
```
Comment (colinrotherham) [inline src/community/contribution-criteria/index.md:80]: This line read a bit funny that published patterns could still be published?

```suggestion
          <p class="govuk-!-margin-bottom-0">Components and patterns published in a limited number of services can still be accepted, as long as it meets other contribution criteria. We’ll be clear about the specific contexts the component or pattern has been tested in and encourage feedback about its performance in a wide range of services.</p>'
```

**Review by colinrotherham — APPROVED** [review]
Looks brill. One suggestion, maybe a little tweak?

File: src/community/contribution-criteria/index.md (hunk 2 above)
Comment (CharlotteDowns) [reply src/community/contribution-criteria/index.md:80]: Thanks Colin, I might go with 'contributed' instead :)

Comment (colinrotherham) [reply src/community/contribution-criteria/index.md]: Fine by me. Press that merge button 🎉

**Pull request merged**